tournaments/index.js

```diff
@@ -1010,7 +1010,7 @@ let commands = {
 			if (Monitor.countPrepBattle(connection.ip, connection)) {
 				return;
 			}
-			TeamValidator(tournament.teambuilderFormat).prepTeam(user.team).then(result => {
+			TeamValidator(tournament.teambuilderFormat, tournament.banlist).prepTeam(user.team).then(result => {
```

Comment:
We also need to pass the banlist to the battles again for ruleset changes, see https://github.com/Zarel/Pokemon-Showdown/commit/fa1b45f7ab19cd420c28bc061e5ae14d4fd40df1.